runio.js

```diff
@@ -0,0 +1,195 @@
+        cfg.replace(/\(optional, default.*?\)/gm, '');
+        cfg.replace(/\\*/gm, '');
+      });
+
+      replaceInFile(`docs/helpers/${name}.md`, cfg => {
+        for (const i in sharedPlaceholders) {
+          cfg.replace(sharedPlaceholders[i], sharedTemplates[i]);
+        }
+      });
+
+      await writeToFile(`docs/helpers/${name}.md`, cfg => {
+        cfg.append(`---\nid: ${name}\ntitle: ${name}\n---\n\n`);
+        cfg.textFromFile(`docs/helpers/${name}.md`);
+      })
+    }
+  },
+
+  async wiki() {
+    // publish wiki pages to website
+    if (!fs.existsSync('website/wiki/Home.md')) {
+      await git(fn => {
+        fn.cloneShallow('git@github.com:Codeception/CodeceptJS.wiki.git', 'website/wiki');
+      });
+    }
+    await chdir('website/wiki', () => git(cfg => cfg.pull('origin master')));
+
+    await writeToFile('docs/community-helpers.md', cfg => {
+      cfg.line('---')
+      cfg.line('id: community-helpers')
+      cfg.line('title: Community Helpers')
+      cfg.line('---')
+      cfg.line('')
+      cfg.line('> Share your helpers at our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Community-Helpers)')
+      cfg.line('')
+      cfg.textFromFile('website/wiki/Community-Helpers.md')
+    });
+
+    writeToFile('docs/examples.md', cfg => {
+      cfg.line('---')
+      cfg.line('id: examples')
+      cfg.line('title: Examples')
+      cfg.line('---')
+      cfg.line('')
+      cfg.line('> Add your own examples to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Examples)')
+      cfg.textFromFile('website/wiki/Examples.md')
+    });
+
+    writeToFile('docs/books.md', cfg => {
+      cfg.line('---')
+      cfg.line('id: books')
+      cfg.line('title: Books & Posts')
+      cfg.line('---')
+      cfg.line('')
+      cfg.line('> Add your own books or posts to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Books-&-Posts)')
+      cfg.textFromFile('website/wiki/Books-&-Posts.md')
+    });
+
+    writeToFile('docs/videos.md', cfg => {
+      cfg.line('---')
+      cfg.line('id: videos')
+      cfg.line('title: Videos')
+      cfg.line('---')
+      cfg.line('')
+      cfg.line('> Add your own videos to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Videos)')
+      cfg.textFromFile('website/wiki/Videos.md')
+    });
+  },
+
+  async publishSite(user = 'davertmik') {
+    // updates codecept.io website
+    await processChangelog();
+    copy('docker/README.md', 'docs/docker.md');
+    await this.wiki();
+    await chdir('website', async () => {
+      await exec('npm install');
+      await npmRun('publish-gh-pages', cfg => {
+        cfg.env('USE_SSH', 'true');
+        cfg.env('GIT_USER', process.env.GIT_USER || user);
+      });
+    });
+  },
+
+
+  async server() {
+    // run test server
+    await Promise.all([
+      exec('php -S 127.0.0.1:8000 -t test/data/app'),
+      npmRun('json-server'),
+      ]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 6. (indent)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4496293)